vikit/prompt/recorded_prompt_subtitles_extractor.py

```
@@ -77,9 +77,25 @@ async def extract_subtitles_async(
 
             subtitle_file_path = "_".join(["subSubtitle", str(i), str(end)]) + ".srt"
 
+            if "output" in subs:
+                if "transcription" in subs["output"]:
+                    transcription = subs["output"]["transcription"]
+                else:
+                    transcription = ""
+                    print("Warning: 'transcription' key exists but has no content.")
+            else:
+                transcription = ""
```

here you already know that we will have to let the user know that "No valid transcription available to write to the subtitle file.", which is the message sent back line 94, so why not logging the message once? Also, iw we have no transcription and the buildsettings are asking for a read aloud  prompt, or if we have been provided a recording without prompt text (a rare case, I concede), then eventually the process will fail, so in those cases we should raise an error and log the error too for user feedback.